optuna/_transform.py

```diff
@@ -114,6 +123,13 @@ def transform(self, params: Dict[str, Any]) -> numpy.ndarray:
                 )
                 bound_idx += 1
 
+        if self._transform_0_1:
+            single_mask = self._raw_bounds[:, 0] == self._raw_bounds[:, 1]
+            trans_params[single_mask] = 0.5
+            trans_params[~single_mask] = (trans_params[~single_mask] - self._raw_bounds[:, 0]) / (
+                self._raw_bounds[:, 1] - self._raw_bounds[:, 0]
+            )
```

Comment:
Can we simplify these lines using `np.where`?

Reply:
That would emit warning, and suppressing that warning would make the code complex.

Reply:
```
with warnings.catch_warnings():
    warnings.simplefilter('ignore')
    trans_params = np.where(self._raw_bounds[:, 0] == self._raw_bounds[:, 1], 0.5, (trans_params[~single_mask] - self._raw_bounds[:, 0]) / (
                self._raw_bounds[:, 1] - self._raw_bounds[:, 0]
            )
```
is what I thought.

Reply:
Could you please tell me what kind of warning that is? I applied the following changes, but could not see the warning message.

```diff
diff --git a/optuna/_transform.py b/optuna/_transform.py
index 34d001ee0..d5b9467f2 100644
--- a/optuna/_transform.py
+++ b/optuna/_transform.py
@@ -123,10 +123,9 @@ class _SearchSpaceTransform:

         if self._transform_0_1:
             single_mask = self._raw_bounds[:, 0] == self._raw_bounds[:, 1]
-            trans_params[single_mask] = 0.5
-            trans_params[~single_mask] = (trans_params[~single_mask] - self._raw_bounds[:, 0]) / (
+            trans_params = numpy.where(single_mask, 0.5, (trans_params[~single_mask] - self._raw_bounds[:, 0]) / (
                 self._raw_bounds[:, 1] - self._raw_bounds[:, 0]
-            )
+            ))

         return trans_params
```

```python
import math
import optuna


def objective(trial: optuna.Trial) -> float:
    return sum([
        math.sin(trial.suggest_float('param-{}'.format(i), -10 * i, 10 * i))
        for i in range(10)
    ])


if __name__ == '__main__':
    study = optuna.create_study(sampler=optuna.samplers.CmaEsSampler(seed=1))
    study.optimize(objective, n_trials=100)
    print(study.best_trial)
```

<details>

<summary>Before</summary>

```
$ python cmaes_example.py
[I 2023-03-14 11:53:09,321] A new study created in memory with name: no-name-e54d9778-6e56-440e-a94c-92f133032b8b
[I 2023-03-14 11:53:09,324] Trial 0 finished with value: 3.07455674013866 and parameters: {'param-0': 0.0, 'param-1': -1.6595599059485195, 'param-2': 8.812979737686323, 'param-3': -29.993137510959308, 'param-4': -15.813394189452818, 'param-5': -35.32441091828869, 'param-6': -48.91936862774426, 'param-7': -43.92357040712608, 'param-8': -24.710283673112357, 'param-9': -18.581854638479413}. Best is trial 0 with value: 3.07455674013866.
[I 2023-03-14 11:53:09,742] Trial 1 finished with value: -1.7858462634570162 and parameters: {'param-0': 0.0, 'param-1': 1.7355285641354712, 'param-2': -0.8237355144468168, 'param-3': -4.677466868830528, 'param-4': -13.479435237682079, 'param-5': 2.8972067653848157, 'param-6': 28.795439963265025, 'param-7': -0.8801076420401159, 'param-8': 29.541322860967156, 'param-9': -6.552154327499167}. Best is trial 1 with value: -1.7858462634570162.
[I 2023-03-14 11:53:09,743] Trial 2 finished with value: 2.745846818414178 and parameters: {'param-0': 0.0, 'param-1': -0.9253334465658583, 'param-2': 7.878718787940571, 'param-3': -3.420259358688334, 'param-4': 13.591591256092919, 'param-5': -29.247133696110815, 'param-6': -13.961325985382594, 'param-7': -5.884960320574734, 'param-8': 46.3389175781445, 'param-9': -5.784128469918514}. Best is trial 1 with value: -1.7858462634570162.
[I 2023-03-14 11:53:09,745] Trial 3 finished with value: -2.109211803026433 and parameters: {'param-0': 0.0, 'param-1': 0.9585221305807714, 'param-2': -3.2063745595433204, 'param-3': -1.7252687448771837, 'param-4': 17.293565576108314, 'param-5': 5.389348374436111, 'param-6': -12.538533449989345, 'param-7': -16.200908940181492, 'param-8': -31.25717243425116, 'param-9': -13.650660861037608}. Best is trial 3 with value: -2.109211803026433.
[I 2023-03-14 11:53:09,746] Trial 4 finished with value: 2.6029570386367418 and parameters: {'param-0': 0.0, 'param-1': -0.25150638956368354, 'param-2': -4.14043907343385, 'param-3': 3.427779040186266, 'param-4': -12.220655997119959, 'param-5': -18.375447524327505, 'param-6': -9.512501969141347, 'param-7': 7.483647015942935, 'param-8': 16.096731202572954, 'param-9': 7.312770173353513}. Best is trial 3 with value: -2.109211803026433.
[I 2023-03-14 11:53:09,747] Trial 5 finished with value: -1.570492385042078 and parameters: {'param-0': 0.0, 'param-1': -0.2684643112500762, 'param-2': 1.2434580643184248, 'param-3': -8.51060115105659, 'param-4': 9.83094620958746, 'param-5': -19.97431020212802, 'param-6': -22.337468348100444, 'param-7': -14.656031362732477, 'param-8': -11.41589798529266, 'param-9': -25.71132773106956}. Best is trial 3 with value: -2.109211803026433.
[I 2023-03-14 11:53:09,748] Trial 6 finished with value: 0.29158593437716473 and parameters: {'param-0': 0.0, 'param-1': -5.435796409444674, 'param-2': 2.0026860676434737, 'param-3': 9.162678326293154, 'param-4': 21.857467085398042, 'param-5': 5.283377046505521, 'param-6': 12.368526255556574, 'param-7': -27.328440258616567, 'param-8': -0.6397425900972848, 'param-9': -5.525289314196343}. Best is trial 3 with value: -2.109211803026433.
[I 2023-03-14 11:53:09,749] Trial 7 finished with value: -0.0007886186524036498 and parameters: {'param-0': 0.0, 'param-1': 5.372035107818158, 'param-2': 1.6780883981446664, 'param-3': -16.408204447093713, 'param-4': 10.342367037901369, 'param-5': -12.234141249743999, 'param-6': -12.319802341995732, 'param-7': 42.14878873531521, 'param-8': -17.459905627005945, 'param-9': 16.40672490190383}. Best is trial 3 with value: -2.109211803026433.
[I 2023-03-14 11:53:09,751] Trial 8 finished with value: -2.8720687849693523 and parameters: {'param-0': 0.0, 'param-1': 2.1169040039981883, 'param-2': 2.8292526326263925, 'param-3': -1.51927036247071, 'param-4': -1.4000382129129179, 'param-5': -16.69598006218125, 'param-6': -8.416848753694026, 'param-7': 4.321351549048657, 'param-8': -19.794183448907404, 'param-9': 16.017054649236798}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:53:09,752] Trial 9 finished with value: 4.030224408029646 and parameters: {'param-0': 0.0, 'param-1': -2.0404262953561254, 'param-2': 1.7173456328074543, 'param-3': -3.835520818279587, 'param-4': -12.28113347819751, 'param-5': -35.87612575809066, 'param-6': -4.541829108064903, 'param-7': -17.79087958397637, 'param-8': -27.71580349905146, 'param-9': -3.9398016471463677}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:53:09,753] Trial 10 finished with value: 1.9631994817050404 and parameters: {'param-0': 0.0, 'param-1': -2.039565618956889, 'param-2': 0.9723904310145102, 'param-3': -2.3064999464712663, 'param-4': 23.576614912671786, 'param-5': -22.60057898220758, 'param-6': -24.733222454390592, 'param-7': 7.732530098678026, 'param-8': -4.662026052710431, 'param-9': 21.033919074926132}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:53:09,755] Trial 11 finished with value: -2.778491696335891 and parameters: {'param-0': 0.0, 'param-1': 2.6751281798844637, 'param-2': -8.215853034538835, 'param-3': -8.291013059372759, 'param-4': 21.42512938705704, 'param-5': -21.199387933110277, 'param-6': -25.184908136425832, 'param-7': -12.643333253651427, 'param-8': -14.773456950284341, 'param-9': 12.284321755877855}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:53:09,758] Trial 12 finished with value: 0.17018850984497835 and parameters: {'param-0': 0.0, 'param-1': 2.820250548622713, 'param-2': 1.136621132622082, 'param-3': -18.70078995723447, 'param-4': -2.5835103508239783, 'param-5': -15.656656204763408, 'param-6': -8.929533505591408, 'param-7': -21.2709434142593, 'param-8': 2.938767183125293, 'param-9': 2.8231308602068026}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:53:09,759] Trial 13 finished with value: -0.20388749212887275 and parameters: {'param-0': 0.0, 'param-1': -0.5802830844558073, 'param-2': -6.350542149128344, 'param-3': 8.952480590579647, 'param-4': 18.749449901557647, 'param-5': -5.167027010135378, 'param-6': -36.692872411681286, 'param-7': -21.415132698691927, 'param-8': -15.231471097579231, 'param-9': -7.035333748090224}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:53:09,760] Trial 14 finished with value: -0.6420108860543067 and parameters: {'param-0': 0.0, 'param-1': 0.4318607173574396, 'param-2': 3.0346406854811008, 'param-3': 12.682998964110276, 'param-4': -3.105492831709576, 'param-5': -3.0641284280965664, 'param-6': -18.41507108243468, 'param-7': -6.504299829394988, 'param-8': -13.57790563957849, 'param-9': 41.392461842213066}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:53:09,761] Trial 15 finished with value: -0.2452110059050675 and parameters: {'param-0': 0.0, 'param-1': 0.3714022746626213, 'param-2': -8.285972464171019, 'param-3': 15.222200357815431, 'param-4': 20.651394305116497, 'param-5': 4.773532839998808, 'param-6': 10.439415555101974, 'param-7': -9.737570017472535, 'param-8': -44.6239604206036, 'param-9': 7.789503427314131}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:53:09,763] Trial 16 finished with value: 2.232056087460125 and parameters: {'param-0': 0.0, 'param-1': 2.2297768246448353, 'param-2': 3.6444783057891073, 'param-3': 3.1381758476708015, 'param-4': 3.857392757435534, 'param-5': -4.113516720963894, 'param-6': 21.209993966533304, 'param-7': 2.6019302779843088, 'param-8': -10.368724859210417, 'param-9': 37.4178189452158}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:53:09,764] Trial 17 finished with value: -0.4812803662504538 and parameters: {'param-0': 0.0, 'param-1': 0.7277089559745242, 'param-2': 9.947966728388455, 'param-3': 2.8432363686265134, 'param-4': -4.552111487391798, 'param-5': -7.504791257110675, 'param-6': -7.2932428962271345, 'param-7': 9.578323851145953, 'param-8': -23.898730304295547, 'param-9': -20.05026212516124}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:53:09,765] Trial 18 finished with value: 0.047321388279854104 and parameters: {'param-0': 0.0, 'param-1': -1.2125147508676193, 'param-2': -2.1591962227639847, 'param-3': 2.315998220031517, 'param-4': 2.9400180870986645, 'param-5': 2.908785719186767, 'param-6': 13.552153657442247, 'param-7': 3.5627531234298147, 'param-8': -2.4280587221328886, 'param-9': 45.05690570072707}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:53:09,766] Trial 19 finished with value: -1.3032210192580413 and parameters: {'param-0': 0.0, 'param-1': 5.716801624948626, 'param-2': 13.308685548548105, 'param-3': -11.619608761773005, 'param-4': -9.120728282140437, 'param-5': -6.090216269562603, 'param-6': 2.9019677989902064, 'param-7': -14.741198176206382, 'param-8': -33.15637558771804, 'param-9': -8.811942191040899}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:53:09,767] Trial 20 finished with value: -3.0656270494144815 and parameters: {'param-0': 0.0, 'param-1': 1.7901250397623123, 'param-2': -7.381056342096175, 'param-3': 16.337029234212295, 'param-4': 2.7769040048445817, 'param-5': -1.0701655822877996, 'param-6': -31.39691693021981, 'param-7': -0.8264426331720074, 'param-8': -13.06776594030977, 'param-9': 24.125703267046717}. Best is trial 20 with value: -3.0656270494144815.
[I 2023-03-14 11:53:09,769] Trial 21 finished with value: -1.145913094787663 and parameters: {'param-0': 0.0, 'param-1': 7.776151262324458, 'param-2': -6.383375695029896, 'param-3': -4.077546497603876, 'param-4': 16.304833190230795, 'param-5': -14.029069051698876, 'param-6': -15.304255817715465, 'param-7': -3.4528726454930165, 'param-8': -12.77855294449489, 'param-9': 29.948503042572653}. Best is trial 20 with value: -3.0656270494144815.
[I 2023-03-14 11:53:09,770] Trial 22 finished with value: -0.6467182827410254 and parameters: {'param-0': 0.0, 'param-1': 3.9054363530918614, 'param-2': -9.657466225589124, 'param-3': 12.875575662416814, 'param-4': -0.6348144603967043, 'param-5': -6.107021719056547, 'param-6': -21.267321455608503, 'param-7': -61.30246260051419, 'param-8': -35.07614598405834, 'param-9': 17.718854991951986}. Best is trial 20 with value: -3.0656270494144815.
[I 2023-03-14 11:53:09,772] Trial 23 finished with value: 2.405729114837162 and parameters: {'param-0': 0.0, 'param-1': 3.9262068176931137, 'param-2': 0.7567369952069747, 'param-3': 1.268906140074403, 'param-4': 5.855977055174861, 'param-5': -6.126436304698849, 'param-6': -21.84383721936448, 'param-7': 15.55764354537473, 'param-8': -5.2208401853942235, 'param-9': 14.68682826810064}. Best is trial 20 with value: -3.0656270494144815.
[I 2023-03-14 11:53:09,773] Trial 24 finished with value: -0.9526879226435986 and parameters: {'param-0': 0.0, 'param-1': 7.101724698514079, 'param-2': 1.2340173907924488, 'param-3': -12.27065120579988, 'param-4': 4.770571155011247, 'param-5': -14.47770613614474, 'param-6': 19.021340821822648, 'param-7': -2.031048819682951, 'param-8': -11.889474701463229, 'param-9': 10.497665426551706}. Best is trial 20 with value: -3.0656270494144815.
[I 2023-03-14 11:53:09,774] Trial 25 finished with value: -1.1518639936432145 and parameters: {'param-0': 0.0, 'param-1': 0.005131629437721941, 'param-2': -12.513806642779745, 'param-3': 16.016339122099197, 'param-4': 9.427955829556709, 'param-5': -8.355269744163927, 'param-6': 14.095940879965639, 'param-7': 16.207078471066154, 'param-8': -24.97044481432077, 'param-9': 30.629309619413462}. Best is trial 20 with value: -3.0656270494144815.
[I 2023-03-14 11:53:09,775] Trial 26 finished with value: -3.134460884130972 and parameters: {'param-0': 0.0, 'param-1': 6.592834121288128, 'param-2': -7.441506435869496, 'param-3': 4.92245071560945, 'param-4': 11.206491681441278, 'param-5': -9.28443768790622, 'param-6': -5.775372709807506, 'param-7': -12.019228638868078, 'param-8': -38.86449629211741, 'param-9': 16.248844798250957}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,776] Trial 27 finished with value: 3.6501567537560287 and parameters: {'param-0': 0.0, 'param-1': 2.549654138102838, 'param-2': -4.962421385204687, 'param-3': -0.6175258917885103, 'param-4': -5.790270348561606, 'param-5': -16.991859528073185, 'param-6': -33.71742830285568, 'param-7': -3.592308171187497, 'param-8': -17.779097604516025, 'param-9': 21.213433218019745}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,777] Trial 28 finished with value: -0.4789539893636976 and parameters: {'param-0': 0.0, 'param-1': -0.8509925725165619, 'param-2': -3.9278102650774933, 'param-3': -3.4457509742058434, 'param-4': 15.441110651553132, 'param-5': -9.447019860255232, 'param-6': -34.25876225233458, 'param-7': -39.45215901163509, 'param-8': -16.96259166716424, 'param-9': 24.367383327602937}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,778] Trial 29 finished with value: 1.1990784520649 and parameters: {'param-0': 0.0, 'param-1': -0.8834132245838298, 'param-2': -13.172459649644402, 'param-3': 15.63285505816691, 'param-4': -10.753980437550162, 'param-5': 6.515376930144768, 'param-6': -24.640816186077913, 'param-7': 14.951236761717453, 'param-8': -13.367560006314662, 'param-9': 14.738018729657114}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,779] Trial 30 finished with value: 4.143212362530454 and parameters: {'param-0': 0.0, 'param-1': 1.0307813407929416, 'param-2': -4.9435600784079305, 'param-3': 7.152550868081796, 'param-4': 7.879449862871283, 'param-5': -1.9388300412021238, 'param-6': 2.6489265804364877, 'param-7': -25.55709555521706, 'param-8': -43.52996333113714, 'param-9': -4.52906901455961}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,781] Trial 31 finished with value: 3.3390752099083416 and parameters: {'param-0': 0.0, 'param-1': 0.9466647414565443, 'param-2': -10.676907728025359, 'param-3': 12.140283980262232, 'param-4': 20.964796086054385, 'param-5': 11.487564133870663, 'param-6': 14.321442218340863, 'param-7': 18.704897731125058, 'param-8': 15.313141412109431, 'param-9': 2.2247562703279016}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,782] Trial 32 finished with value: -2.425339801313445 and parameters: {'param-0': 0.0, 'param-1': 9.013091512310542, 'param-2': -11.408528090777535, 'param-3': 4.629593919034569, 'param-4': 23.404656491339175, 'param-5': -7.436219667473125, 'param-6': -6.5446582784887255, 'param-7': -17.54900227598369, 'param-8': -7.475462317906306, 'param-9': 41.50842145887526}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,783] Trial 33 finished with value: -1.4435452672025417 and parameters: {'param-0': 0.0, 'param-1': 5.463330556826309, 'param-2': -6.413513406378035, 'param-3': 8.161716023744674, 'param-4': 8.19429106219011, 'param-5': -8.756506958248636, 'param-6': -1.7266126937545963, 'param-7': 21.347228107288515, 'param-8': -21.013263767571424, 'param-9': 30.719193037282153}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,784] Trial 34 finished with value: 4.718080694996887 and parameters: {'param-0': 0.0, 'param-1': 1.589042172768071, 'param-2': -4.376895472601664, 'param-3': 16.84459973638984, 'param-4': 13.52784651187497, 'param-5': -10.52361642563401, 'param-6': 26.503058228495917, 'param-7': -15.821502375770578, 'param-8': -35.46514358698461, 'param-9': 18.93887050482715}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,785] Trial 35 finished with value: 4.532316522957611 and parameters: {'param-0': 0.0, 'param-1': 3.966186271929651, 'param-2': -4.8528642597610485, 'param-3': 2.582225267459627, 'param-4': 14.488880810638953, 'param-5': 2.4377309966482557, 'param-6': 5.791207788693214, 'param-7': -61.99840454159726, 'param-8': -48.416120138018904, 'param-9': 71.06084601690955}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,786] Trial 36 finished with value: 0.3366194971726634 and parameters: {'param-0': 0.0, 'param-1': 6.35373859640908, 'param-2': -12.651928669407948, 'param-3': 9.014317097878532, 'param-4': -8.208015923794466, 'param-5': -23.481402808361196, 'param-6': 25.98650823993279, 'param-7': 9.965441235129674, 'param-8': -14.2930980729585, 'param-9': 0.6978618444820768}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,788] Trial 37 finished with value: -1.6210378540684625 and parameters: {'param-0': 0.0, 'param-1': 6.098488452745226, 'param-2': -16.175882211335143, 'param-3': 1.0966054187318868, 'param-4': 10.690419163333225, 'param-5': -14.38244316817621, 'param-6': 7.412444054528649, 'param-7': -0.2508406064023774, 'param-8': -45.19063822215112, 'param-9': -2.528916060955922}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,789] Trial 38 finished with value: -0.7567122280673153 and parameters: {'param-0': 0.0, 'param-1': 5.556343675056299, 'param-2': -8.890982496712958, 'param-3': -6.648188198364554, 'param-4': 3.114566157710442, 'param-5': -16.381976757278437, 'param-6': -8.663294402662444, 'param-7': 19.485002014485445, 'param-8': -25.894668343706982, 'param-9': 13.707689309189817}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,790] Trial 39 finished with value: 0.33972907818342113 and parameters: {'param-0': 0.0, 'param-1': 1.4906947655591125, 'param-2': -12.865456743864492, 'param-3': 3.0822933955022904, 'param-4': 16.746615467467088, 'param-5': -5.53313986017907, 'param-6': 8.387575153138329, 'param-7': -11.92276512221099, 'param-8': 11.163083096852873, 'param-9': -2.3425367260051217}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,791] Trial 40 finished with value: -1.8299997521511608 and parameters: {'param-0': 0.0, 'param-1': 0.08475465774924018, 'param-2': -6.2916454857255815, 'param-3': -13.854218821453973, 'param-4': 8.52979460313832, 'param-5': -12.817475202423154, 'param-6': -19.035366565043056, 'param-7': -2.6313115215011322, 'param-8': -33.637535863857195, 'param-9': 50.25655874668203}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,793] Trial 41 finished with value: 3.6194297091780525 and parameters: {'param-0': 0.0, 'param-1': 1.9470516977860726, 'param-2': -4.641425079142428, 'param-3': 6.136426033471523, 'param-4': 20.294216233063693, 'param-5': -19.08989816796688, 'param-6': 1.4379608733113898, 'param-7': -14.753845470522606, 'param-8': -3.3294453904637464, 'param-9': 38.5057655500579}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,795] Trial 42 finished with value: 2.5014591759487006 and parameters: {'param-0': 0.0, 'param-1': 5.935469745073348, 'param-2': -11.481487558793233, 'param-3': 7.151071021525503, 'param-4': 14.162662679294328, 'param-5': -14.203848866093516, 'param-6': -12.669486456278612, 'param-7': -23.412815341297545, 'param-8': -34.23455464619374, 'param-9': 25.807247384941576}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,796] Trial 43 finished with value: 4.173215105985346 and parameters: {'param-0': 0.0, 'param-1': -0.2753837085702564, 'param-2': -4.708208420148875, 'param-3': 7.668854403660433, 'param-4': 18.513069999582605, 'param-5': -5.882827578083635, 'param-6': -30.24003190726989, 'param-7': 20.48522051145011, 'param-8': -60.273279658668415, 'param-9': 87.89571328043417}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,797] Trial 44 finished with value: 1.847169722157606 and parameters: {'param-0': 0.0, 'param-1': 3.331930612375091, 'param-2': -10.865424872055403, 'param-3': -0.055690575607478365, 'param-4': 10.406526749222415, 'param-5': 2.6230231833256994, 'param-6': -5.693899200997478, 'param-7': 6.442190591256136, 'param-8': 6.285418407597746, 'param-9': 2.33782650883154}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,798] Trial 45 finished with value: 5.261282849921536 and parameters: {'param-0': 0.0, 'param-1': 1.2045651008038032, 'param-2': -10.7154683485313, 'param-3': 1.851515678227475, 'param-4': 0.4682189183140508, 'param-5': -18.314240048251463, 'param-6': -4.863941805050153, 'param-7': 46.713624774104815, 'param-8': -49.58560043222555, 'param-9': 24.52406110499929}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,815] Trial 46 finished with value: -2.1244373959350953 and parameters: {'param-0': 0.0, 'param-1': 5.864348145135281, 'param-2': -13.686330749798525, 'param-3': 4.706353299399403, 'param-4': 8.064102615308606, 'param-5': -38.44576874736448, 'param-6': 6.0554842240631075, 'param-7': -43.52348448727592, 'param-8': -40.45224464451792, 'param-9': 62.87699556497827}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,817] Trial 47 finished with value: -1.123613716955029 and parameters: {'param-0': 0.0, 'param-1': 4.315221999106969, 'param-2': -6.5428640123902255, 'param-3': -6.96394911958874, 'param-4': 3.6257923830902996, 'param-5': -30.049206159278043, 'param-6': -9.75935965318532, 'param-7': -40.98223086546052, 'param-8': -6.089303801693916, 'param-9': 66.48684649484602}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,818] Trial 48 finished with value: -0.3394014294403014 and parameters: {'param-0': 0.0, 'param-1': 5.985944833599383, 'param-2': -10.717615131367658, 'param-3': -2.9855305058545625, 'param-4': -3.206853771849481, 'param-5': -20.947041312392088, 'param-6': -6.263562514039194, 'param-7': -34.02852436076248, 'param-8': -54.261104610430294, 'param-9': 37.370917023991396}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,819] Trial 49 finished with value: -0.5755668207908291 and parameters: {'param-0': 0.0, 'param-1': 5.884150717640708, 'param-2': -11.27383543580115, 'param-3': 1.6934382851061898, 'param-4': 8.733265539153905, 'param-5': -7.2527835813591395, 'param-6': -10.105406933389602, 'param-7': 4.1119241401215305, 'param-8': -27.048533533224003, 'param-9': 30.459325028225663}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,820] Trial 50 finished with value: 0.002732648501080681 and parameters: {'param-0': 0.0, 'param-1': 8.745771354722326, 'param-2': -8.638170359437769, 'param-3': 0.1302850284261936, 'param-4': 32.86215749768911, 'param-5': -20.502456087810074, 'param-6': -1.7307186109503547, 'param-7': -4.274989071404917, 'param-8': -42.584171272945206, 'param-9': 30.173376403993075}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,822] Trial 51 finished with value: 1.9648463443162272 and parameters: {'param-0': 0.0, 'param-1': 5.202816051830332, 'param-2': 4.322365011158237, 'param-3': 19.34213602153627, 'param-4': 14.326884381310123, 'param-5': -23.5833362979602, 'param-6': 9.373053553780963, 'param-7': -48.39435497959147, 'param-8': -4.391630033846155, 'param-9': 16.400885013865874}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,823] Trial 52 finished with value: 3.4229635858181666 and parameters: {'param-0': 0.0, 'param-1': 3.823733617807372, 'param-2': -11.635259881682835, 'param-3': 8.733299876530097, 'param-4': 14.163704421564049, 'param-5': -17.1737858882587, 'param-6': -8.71017383650144, 'param-7': -5.364468846497445, 'param-8': -8.900600208519478, 'param-9': 39.071153191978254}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,824] Trial 53 finished with value: 1.3993851028421667 and parameters: {'param-0': 0.0, 'param-1': 8.482107403792494, 'param-2': -6.867894801203757, 'param-3': 6.3119427637875205, 'param-4': 17.921146149068207, 'param-5': -42.18276509319164, 'param-6': -4.5562195768071945, 'param-7': -50.14464249454794, 'param-8': -20.35950343435139, 'param-9': 83.84399121011131}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,825] Trial 54 finished with value: 0.9513765703765935 and parameters: {'param-0': 0.0, 'param-1': 2.406188702488702, 'param-2': -9.633393452611113, 'param-3': 8.195297353, 'param-4': 18.80447908479873, 'param-5': -32.031298001373976, 'param-6': 10.782246356774735, 'param-7': -35.50647851490283, 'param-8': -48.285153685094016, 'param-9': 67.57910140656253}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,826] Trial 55 finished with value: -2.1816017280899436 and parameters: {'param-0': 0.0, 'param-1': 2.954706344728743, 'param-2': -7.55287557244249, 'param-3': -1.403183396095379, 'param-4': 4.341784362631458, 'param-5': -24.682656189333578, 'param-6': -19.046490898434854, 'param-7': -36.405306909886306, 'param-8': -13.050269029986922, 'param-9': 41.07311645330026}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,827] Trial 56 finished with value: -0.40149755512636975 and parameters: {'param-0': 0.0, 'param-1': 3.5866088201478306, 'param-2': -12.151484433594991, 'param-3': -12.148264164925425, 'param-4': -2.3052166627420974, 'param-5': -19.394579466929518, 'param-6': 20.032984614639943, 'param-7': -69.12605257777267, 'param-8': 16.149511740060547, 'param-9': 84.83003846870363}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,829] Trial 57 finished with value: -0.5793471908897462 and parameters: {'param-0': 0.0, 'param-1': 3.7220420736493374, 'param-2': 0.07358910366919957, 'param-3': -13.8459590842101, 'param-4': 12.56478962108745, 'param-5': -23.068141676165343, 'param-6': 2.508958145208858, 'param-7': -15.803421181256795, 'param-8': -50.23384553662599, 'param-9': 41.67935845331712}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,830] Trial 58 finished with value: 1.5924883802380394 and parameters: {'param-0': 0.0, 'param-1': 3.085793471429639, 'param-2': -9.469237110817204, 'param-3': -5.212688179304106, 'param-4': 1.4231761059741288, 'param-5': -32.44014447825038, 'param-6': 12.359431600931288, 'param-7': -33.36425122351743, 'param-8': -35.40496412132269, 'param-9': 32.46201581459228}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,831] Trial 59 finished with value: 1.9569231942341943 and parameters: {'param-0': 0.0, 'param-1': 1.1243046872353837, 'param-2': -4.143971844466057, 'param-3': 13.221529936144847, 'param-4': -14.764329090715115, 'param-5': -35.511616937940595, 'param-6': 0.14236046683453907, 'param-7': -65.35171968280288, 'param-8': -33.927779484723416, 'param-9': 77.86300136769964}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,832] Trial 60 finished with value: 4.584185380779558 and parameters: {'param-0': 0.0, 'param-1': 0.8031151512522108, 'param-2': -11.688194015792952, 'param-3': 7.506711972428448, 'param-4': -11.337010023340461, 'param-5': -24.903275838154947, 'param-6': 31.584785616852898, 'param-7': -43.03331326406659, 'param-8': -46.73509629438097, 'param-9': 27.88103255135617}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:53:09,834] Trial 61 finished with value: -3.7024967182018313 and parameters: {'param-0': 0.0, 'param-1': -1.5783790085883638, 'param-2': 4.419564712851731, 'param-3': 1.8592947852869415, 'param-4': 4.411727358218435, 'param-5': -6.811200421880557, 'param-6': -43.70787707154304, 'param-7': -14.716110244675079, 'param-8': -28.511051234991726, 'param-9': 35.708747313333134}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:53:09,835] Trial 62 finished with value: -0.16937136132681896 and parameters: {'param-0': 0.0, 'param-1': 6.046255509790683, 'param-2': -14.121530769280138, 'param-3': 7.664462877352008, 'param-4': 11.857795148633336, 'param-5': -30.640851786811258, 'param-6': 20.408464559345816, 'param-7': -16.78794654141717, 'param-8': -7.38522188380837, 'param-9': 29.546145225782894}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:53:09,837] Trial 63 finished with value: 0.16504466380504695 and parameters: {'param-0': 0.0, 'param-1': 3.911434040991635, 'param-2': -10.961143693967593, 'param-3': -2.2752816048310613, 'param-4': 28.74189414589152, 'param-5': -25.172392920267352, 'param-6': 6.90425076123536, 'param-7': -24.9596432963433, 'param-8': -41.31498747960013, 'param-9': 72.35366596061579}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:53:09,838] Trial 64 finished with value: -1.6531747253503348 and parameters: {'param-0': 0.0, 'param-1': 2.8556033157794793, 'param-2': -2.499422414068185, 'param-3': -15.75229708241375, 'param-4': 4.6469184137162145, 'param-5': -12.64807383826416, 'param-6': -6.631669534753968, 'param-7': -12.679793042922348, 'param-8': -6.902471441660666, 'param-9': 82.50547085728493}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:53:09,840] Trial 65 finished with value: 0.4813406168180178 and parameters: {'param-0': 0.0, 'param-1': 2.610927564880626, 'param-2': -13.723392423599428, 'param-3': -14.76552966730726, 'param-4': 11.9971460749613, 'param-5': -29.985755791811712, 'param-6': -2.3537847377259027, 'param-7': -67.61139644872165, 'param-8': -12.144457632858632, 'param-9': 71.67397811224728}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:53:09,841] Trial 66 finished with value: -0.3719758239424421 and parameters: {'param-0': 0.0, 'param-1': 8.408761779183386, 'param-2': -1.4187575370889007, 'param-3': -11.099459165650586, 'param-4': 30.505594660665707, 'param-5': -3.1368910646616754, 'param-6': -18.797863800298963, 'param-7': 17.48318396888486, 'param-8': -36.16625735269997, 'param-9': 5.753225103478343}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:53:09,842] Trial 67 finished with value: -0.6806850044923272 and parameters: {'param-0': 0.0, 'param-1': 4.114647420676874, 'param-2': 1.8509339057588647, 'param-3': 5.471167876711725, 'param-4': 16.550970516643204, 'param-5': -16.21615223768432, 'param-6': -15.863760403591108, 'param-7': -22.003333012008476, 'param-8': -30.976201710522197, 'param-9': 24.696640756859665}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:53:09,843] Trial 68 finished with value: -2.0421144402028255 and parameters: {'param-0': 0.0, 'param-1': -1.4099153907576287, 'param-2': 6.587608147363262, 'param-3': -10.845938068054327, 'param-4': -9.140522118162142, 'param-5': -21.825039216688342, 'param-6': 4.519774791328359, 'param-7': -7.681845586645018, 'param-8': -26.324898874278105, 'param-9': 39.208106888847965}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:53:09,844] Trial 69 finished with value: 0.8928730621682152 and parameters: {'param-0': 0.0, 'param-1': -3.2436970362314597, 'param-2': 1.3245764940515485, 'param-3': -11.572998728949692, 'param-4': 1.7794548970201944, 'param-5': -23.261399066709505, 'param-6': -14.659597140704939, 'param-7': -33.12186296399628, 'param-8': -38.11657085725801, 'param-9': 62.07405984524581}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:53:09,845] Trial 70 finished with value: 3.995776158146804 and parameters: {'param-0': 0.0, 'param-1': 1.9289330280648294, 'param-2': -3.2832722821513904, 'param-3': 1.5522265934425974, 'param-4': -3.3794323706385256, 'param-5': -34.45119265176209, 'param-6': 7.875605414935563, 'param-7': -54.28455791709932, 'param-8': -32.757976742473446, 'param-9': 70.57411760551881}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:53:09,847] Trial 71 finished with value: 1.150482656681782 and parameters: {'param-0': 0.0, 'param-1': -0.026055278004466942, 'param-2': -8.428558021262194, 'param-3': -11.54623864249799, 'param-4': 1.765290656711251, 'param-5': -23.482973035606957, 'param-6': -31.296504617201396, 'param-7': -24.74298046238207, 'param-8': -15.268217302876408, 'param-9': 55.456463610953676}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:53:09,848] Trial 72 finished with value: -1.6428969427737077 and parameters: {'param-0': 0.0, 'param-1': 1.3467127760112412, 'param-2': 7.1078063341415, 'param-3': -15.749413627569902, 'param-4': -1.7444442361408221, 'param-5': -25.17598303438454, 'param-6': 5.189507244166677, 'param-7': -13.864384648915085, 'param-8': -35.0579368939648, 'param-9': 73.93811972230861}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:53:09,849] Trial 73 finished with value: 1.8621210646303208 and parameters: {'param-0': 0.0, 'param-1': 2.76848129390282, 'param-2': 10.849910938153396, 'param-3': 1.027852910013575, 'param-4': -17.004658673250994, 'param-5': -3.185381580907496, 'param-6': -36.20662651355284, 'param-7': 6.854135848962372, 'param-8': -15.77446043237525, 'param-9': 23.76510132890904}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:53:09,850] Trial 74 finished with value: 0.3832508101514548 and parameters: {'param-0': 0.0, 'param-1': -2.8886746038330173, 'param-2': 8.616880056174303, 'param-3': -13.337528989038681, 'param-4': -9.4538093562782, 'param-5': -26.992664282328363, 'param-6': 13.991815570114085, 'param-7': 18.577086510165202, 'param-8': -22.500245443989144, 'param-9': 82.0170175108239}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:53:09,851] Trial 75 finished with value: 1.674257290500921 and parameters: {'param-0': 0.0, 'param-1': 6.32721513494014, 'param-2': 4.762698353125781, 'param-3': -18.702246252744338, 'param-4': 22.288142111193324, 'param-5': -24.28198788533924, 'param-6': -13.26988237824748, 'param-7': -22.866583480328615, 'param-8': -29.896010228111948, 'param-9': 7.410911781133578}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:53:09,852] Trial 76 finished with value: -1.0468725115633561 and parameters: {'param-0': 0.0, 'param-1': -1.1610687929452332, 'param-2': 6.559642278817542, 'param-3': -3.5598466943229496, 'param-4': 3.487176494093461, 'param-5': -25.33578906649397, 'param-6': -20.165061111008413, 'param-7': -9.740563204066184, 'param-8': -24.53055399066541, 'param-9': 3.320318187915106}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:53:09,853] Trial 77 finished with value: 4.239897508448774 and parameters: {'param-0': 0.0, 'param-1': -0.18785019765469002, 'param-2': 13.715626286308655, 'param-3': -15.82853615457911, 'param-4': 8.481490542106002, 'param-5': -5.695079996898407, 'param-6': -18.982326288019493, 'param-7': -36.276208212765496, 'param-8': -17.52820750515197, 'param-9': 28.069020499189463}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:53:09,854] Trial 78 finished with value: -5.1657150247361985 and parameters: {'param-0': 0.0, 'param-1': -0.6379163450129948, 'param-2': 1.8159126429309147, 'param-3': 11.535288239524945, 'param-4': -13.404787650407659, 'param-5': -27.35103995100866, 'param-6': -13.352429922820598, 'param-7': -46.08319803074357, 'param-8': -33.67106833131295, 'param-9': 49.344649791676716}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:53:09,856] Trial 79 finished with value: -4.387563821089634 and parameters: {'param-0': 0.0, 'param-1': -2.139560424337783, 'param-2': 12.859799939386875, 'param-3': 3.763327352915539, 'param-4': -9.860565074394493, 'param-5': 4.705403191855275, 'param-6': -27.158640878722274, 'param-7': 11.649194924106894, 'param-8': -3.158042552507453, 'param-9': 36.18694644566973}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:53:09,857] Trial 80 finished with value: -2.1077783523767315 and parameters: {'param-0': 0.0, 'param-1': -1.5053462315868984, 'param-2': 16.857382999748253, 'param-3': -14.306758627539198, 'param-4': 18.59178613146804, 'param-5': 8.171723528578092, 'param-6': -56.4698001958446, 'param-7': -20.67325012116806, 'param-8': -4.915177560091394, 'param-9': 50.26862168864079}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:53:09,859] Trial 81 finished with value: 1.9166016850612984 and parameters: {'param-0': 0.0, 'param-1': -3.858454733916627, 'param-2': 7.894883832247327, 'param-3': -8.592362560696085, 'param-4': 17.92006542960928, 'param-5': -44.327603833441174, 'param-6': -16.66324137759125, 'param-7': 20.819584379083167, 'param-8': -12.500507416602773, 'param-9': 63.17465348007366}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:53:09,860] Trial 82 finished with value: 2.537036780052864 and parameters: {'param-0': 0.0, 'param-1': -5.73656727604585, 'param-2': 0.2438272383147222, 'param-3': 6.319557907324416, 'param-4': -10.284486966914823, 'param-5': -24.52419847114674, 'param-6': -17.62674468088587, 'param-7': -27.749249278389875, 'param-8': -0.5757144511673005, 'param-9': 15.16603800028264}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:53:09,861] Trial 83 finished with value: 0.7609992566997936 and parameters: {'param-0': 0.0, 'param-1': 1.3973502503023632, 'param-2': 4.226024932708107, 'param-3': -3.1882079968997026, 'param-4': -0.6927692038259323, 'param-5': -13.254442460067288, 'param-6': -10.473005317457385, 'param-7': -19.61904883959418, 'param-8': -30.224451151777842, 'param-9': 33.650813729567815}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:53:09,862] Trial 84 finished with value: 1.1919141204107087 and parameters: {'param-0': 0.0, 'param-1': -4.370447514239906, 'param-2': 15.520460762938136, 'param-3': -10.217096400294245, 'param-4': -13.7897949110701, 'param-5': -28.768441119002468, 'param-6': -13.127147107825373, 'param-7': -19.25185792181118, 'param-8': -11.930832063671815, 'param-9': 12.714122609292943}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:53:09,863] Trial 85 finished with value: 5.567073184051192 and parameters: {'param-0': 0.0, 'param-1': 1.2656846733472733, 'param-2': 8.376380567771967, 'param-3': -9.387137366272338, 'param-4': -11.546184121692427, 'param-5': 2.06999843103236, 'param-6': -2.426212095006612, 'param-7': -42.103489558181025, 'param-8': -30.22166646321196, 'param-9': 57.52262962206157}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:53:09,864] Trial 86 finished with value: -1.593024093164151 and parameters: {'param-0': 0.0, 'param-1': -0.8286774214642314, 'param-2': 13.24842775938302, 'param-3': 9.975679318305453, 'param-4': -21.494082511681995, 'param-5': -23.843270998969068, 'param-6': -1.718085785648782, 'param-7': -7.344768618814214, 'param-8': -12.739341990138684, 'param-9': 13.194521835755793}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:53:09,865] Trial 87 finished with value: -1.187663197889515 and parameters: {'param-0': 0.0, 'param-1': -2.1604436747293, 'param-2': 6.3785604962960285, 'param-3': -4.664850861119163, 'param-4': -19.127104117357234, 'param-5': -32.90003465530168, 'param-6': -11.835037735609085, 'param-7': -37.435319933210955, 'param-8': -20.484718351100923, 'param-9': 37.587822173137255}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:53:09,867] Trial 88 finished with value: 2.2698279842419447 and parameters: {'param-0': 0.0, 'param-1': 0.14091357621384581, 'param-2': 14.08854416982232, 'param-3': 6.661342938211895, 'param-4': -6.584994298816191, 'param-5': -3.6758127406666716, 'param-6': -43.70390448058216, 'param-7': -4.964533748754235, 'param-8': -45.62040554772884, 'param-9': 38.00797573305198}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:53:09,868] Trial 89 finished with value: -1.0894970962662252 and parameters: {'param-0': 0.0, 'param-1': -0.1985634757039776, 'param-2': 3.3712220734854412, 'param-3': 4.921320588907179, 'param-4': -10.209553629215883, 'param-5': -30.837734578016857, 'param-6': -14.774871086981932, 'param-7': -29.312476792027333, 'param-8': -21.871900735654663, 'param-9': 73.32950294818255}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:53:09,869] Trial 90 finished with value: -0.7892663375873417 and parameters: {'param-0': 0.0, 'param-1': -3.753221602915115, 'param-2': 5.356661491883411, 'param-3': -9.52023187267767, 'param-4': 8.444391579463392, 'param-5': -21.09898592881474, 'param-6': -25.301337831278396, 'param-7': -16.61914964893672, 'param-8': -38.06819007190808, 'param-9': 61.493101807795114}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:53:09,870] Trial 91 finished with value: 0.7900370067341222 and parameters: {'param-0': 0.0, 'param-1': -3.4587831951038206, 'param-2': 17.02193567194964, 'param-3': -18.642231227536584, 'param-4': -34.056504264521166, 'param-5': -23.79367108902618, 'param-6': -19.896844132123903, 'param-7': 1.2361985605482317, 'param-8': -10.686315752274297, 'param-9': 24.844267682439707}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:53:09,872] Trial 92 finished with value: 1.4357065187484959 and parameters: {'param-0': 0.0, 'param-1': 0.18195713040617, 'param-2': 2.5393199904163097, 'param-3': -0.17469399949462172, 'param-4': -19.038578360599093, 'param-5': -20.746242052748514, 'param-6': 3.1545494522133524, 'param-7': -17.178192407398377, 'param-8': -18.675670423702257, 'param-9': 46.12234354023357}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:53:09,873] Trial 93 finished with value: 1.332182331440031 and parameters: {'param-0': 0.0, 'param-1': 2.0370533136664157, 'param-2': -4.350851107825646, 'param-3': 5.008909436261391, 'param-4': 7.7871162592198, 'param-5': -39.6520378555507, 'param-6': 20.303537240732823, 'param-7': -8.265465460834697, 'param-8': 12.739453040397166, 'param-9': 25.274375614492754}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:53:09,874] Trial 94 finished with value: 0.40226422472793383 and parameters: {'param-0': 0.0, 'param-1': -6.1194773685811565, 'param-2': 11.018348416053119, 'param-3': 3.553352741203966, 'param-4': -27.42635063634117, 'param-5': -17.82842579138267, 'param-6': -3.2035363285535894, 'param-7': 21.356894974181685, 'param-8': 19.16180809359338, 'param-9': -30.803547439789}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:53:09,875] Trial 95 finished with value: -1.2565270265317818 and parameters: {'param-0': 0.0, 'param-1': -3.101331149628095, 'param-2': 3.973159512201793, 'param-3': -21.921873775272836, 'param-4': -8.413896914481409, 'param-5': -16.63505371381693, 'param-6': 2.655747153563091, 'param-7': 5.587648258267876, 'param-8': -24.450855339040643, 'param-9': 43.025999162573214}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:53:09,876] Trial 96 finished with value: -0.38334157874526686 and parameters: {'param-0': 0.0, 'param-1': -0.5462971040180911, 'param-2': 13.371927980411712, 'param-3': 22.383131644870637, 'param-4': -37.67515041764983, 'param-5': -25.0370079797484, 'param-6': -39.523577004865786, 'param-7': -15.108133668102809, 'param-8': -61.32390742758892, 'param-9': 37.91264949828205}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:53:09,877] Trial 97 finished with value: 2.143693476406879 and parameters: {'param-0': 0.0, 'param-1': -3.4477349322443356, 'param-2': 12.992862632623805, 'param-3': 2.0464340812718973, 'param-4': -17.817610598207448, 'param-5': -45.484826476283196, 'param-6': -2.097434051166978, 'param-7': -37.46823986652984, 'param-8': 8.68460676366324, 'param-9': 27.579492672096194}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:53:09,878] Trial 98 finished with value: -0.4721133717660624 and parameters: {'param-0': 0.0, 'param-1': -0.6685362691693424, 'param-2': 11.389123387056546, 'param-3': 8.155258288837707, 'param-4': -24.117842099798604, 'param-5': -10.192299404151399, 'param-6': -14.076152750684429, 'param-7': -32.153057098472516, 'param-8': -4.206544169711663, 'param-9': 10.10880727646743}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:53:09,879] Trial 99 finished with value: -1.3476625107916274 and parameters: {'param-0': 0.0, 'param-1': 2.2797704200583198, 'param-2': 15.633170735927983, 'param-3': -7.496188700385282, 'param-4': 4.282449410083096, 'param-5': -26.11404793929163, 'param-6': -10.221254746173926, 'param-7': -39.42921570589232, 'param-8': -18.701178818045364, 'param-9': 2.4727491512137334}. Best is trial 78 with value: -5.1657150247361985.
FrozenTrial(number=78, state=TrialState.COMPLETE, values=[-5.1657150247361985], datetime_start=datetime.datetime(2023, 3, 14, 11, 53, 9, 853987), datetime_complete=datetime.datetime(2023, 3, 14, 11, 53, 9, 854760), params={'param-0': 0.0, 'param-1': -0.6379163450129948, 'param-2': 1.8159126429309147, 'param-3': 11.535288239524945, 'param-4': -13.404787650407659, 'param-5': -27.35103995100866, 'param-6': -13.352429922820598, 'param-7': -46.08319803074357, 'param-8': -33.67106833131295, 'param-9': 49.344649791676716}, user_attrs={}, system_attrs={'cma:generation': 7, 'cma:n_restarts': 0}, intermediate_values={}, distributions={'param-0': FloatDistribution(high=0.0, log=False, low=0.0, step=None), 'param-1': FloatDistribution(high=10.0, log=False, low=-10.0, step=None), 'param-2': FloatDistribution(high=20.0, log=False, low=-20.0, step=None), 'param-3': FloatDistribution(high=30.0, log=False, low=-30.0, step=None), 'param-4': FloatDistribution(high=40.0, log=False, low=-40.0, step=None), 'param-5': FloatDistribution(high=50.0, log=False, low=-50.0, step=None), 'param-6': FloatDistribution(high=60.0, log=False, low=-60.0, step=None), 'param-7': FloatDistribution(high=70.0, log=False, low=-70.0, step=None), 'param-8': FloatDistribution(high=80.0, log=False, low=-80.0, step=None), 'param-9': FloatDistribution(high=90.0, log=False, low=-90.0, step=None)}, trial_id=78, value=None)
```

</details>


<details>

<summary>After</summary>

```
$ python cmaes_example.py
[I 2023-03-14 11:52:52,183] A new study created in memory with name: no-name-2815cbd0-a357-4a6c-b2f2-52826e59beb5
[I 2023-03-14 11:52:52,185] Trial 0 finished with value: 3.07455674013866 and parameters: {'param-0': 0.0, 'param-1': -1.6595599059485195, 'param-2': 8.812979737686323, 'param-3': -29.993137510959308, 'param-4': -15.813394189452818, 'param-5': -35.32441091828869, 'param-6': -48.91936862774426, 'param-7': -43.92357040712608, 'param-8': -24.710283673112357, 'param-9': -18.581854638479413}. Best is trial 0 with value: 3.07455674013866.
[I 2023-03-14 11:52:52,449] Trial 1 finished with value: -1.7858462634570162 and parameters: {'param-0': 0.0, 'param-1': 1.7355285641354712, 'param-2': -0.8237355144468168, 'param-3': -4.677466868830528, 'param-4': -13.479435237682079, 'param-5': 2.8972067653848157, 'param-6': 28.795439963265025, 'param-7': -0.8801076420401159, 'param-8': 29.541322860967156, 'param-9': -6.552154327499167}. Best is trial 1 with value: -1.7858462634570162.
[I 2023-03-14 11:52:52,450] Trial 2 finished with value: 2.745846818414178 and parameters: {'param-0': 0.0, 'param-1': -0.9253334465658583, 'param-2': 7.878718787940571, 'param-3': -3.420259358688334, 'param-4': 13.591591256092919, 'param-5': -29.247133696110815, 'param-6': -13.961325985382594, 'param-7': -5.884960320574734, 'param-8': 46.3389175781445, 'param-9': -5.784128469918514}. Best is trial 1 with value: -1.7858462634570162.
[I 2023-03-14 11:52:52,451] Trial 3 finished with value: -2.109211803026433 and parameters: {'param-0': 0.0, 'param-1': 0.9585221305807714, 'param-2': -3.2063745595433204, 'param-3': -1.7252687448771837, 'param-4': 17.293565576108314, 'param-5': 5.389348374436111, 'param-6': -12.538533449989345, 'param-7': -16.200908940181492, 'param-8': -31.25717243425116, 'param-9': -13.650660861037608}. Best is trial 3 with value: -2.109211803026433.
[I 2023-03-14 11:52:52,453] Trial 4 finished with value: 2.6029570386367418 and parameters: {'param-0': 0.0, 'param-1': -0.25150638956368354, 'param-2': -4.14043907343385, 'param-3': 3.427779040186266, 'param-4': -12.220655997119959, 'param-5': -18.375447524327505, 'param-6': -9.512501969141347, 'param-7': 7.483647015942935, 'param-8': 16.096731202572954, 'param-9': 7.312770173353513}. Best is trial 3 with value: -2.109211803026433.
[I 2023-03-14 11:52:52,454] Trial 5 finished with value: -1.570492385042078 and parameters: {'param-0': 0.0, 'param-1': -0.2684643112500762, 'param-2': 1.2434580643184248, 'param-3': -8.51060115105659, 'param-4': 9.83094620958746, 'param-5': -19.97431020212802, 'param-6': -22.337468348100444, 'param-7': -14.656031362732477, 'param-8': -11.41589798529266, 'param-9': -25.71132773106956}. Best is trial 3 with value: -2.109211803026433.
[I 2023-03-14 11:52:52,455] Trial 6 finished with value: 0.29158593437716473 and parameters: {'param-0': 0.0, 'param-1': -5.435796409444674, 'param-2': 2.0026860676434737, 'param-3': 9.162678326293154, 'param-4': 21.857467085398042, 'param-5': 5.283377046505521, 'param-6': 12.368526255556574, 'param-7': -27.328440258616567, 'param-8': -0.6397425900972848, 'param-9': -5.525289314196343}. Best is trial 3 with value: -2.109211803026433.
[I 2023-03-14 11:52:52,456] Trial 7 finished with value: -0.0007886186524036498 and parameters: {'param-0': 0.0, 'param-1': 5.372035107818158, 'param-2': 1.6780883981446664, 'param-3': -16.408204447093713, 'param-4': 10.342367037901369, 'param-5': -12.234141249743999, 'param-6': -12.319802341995732, 'param-7': 42.14878873531521, 'param-8': -17.459905627005945, 'param-9': 16.40672490190383}. Best is trial 3 with value: -2.109211803026433.
[I 2023-03-14 11:52:52,458] Trial 8 finished with value: -2.8720687849693523 and parameters: {'param-0': 0.0, 'param-1': 2.1169040039981883, 'param-2': 2.8292526326263925, 'param-3': -1.51927036247071, 'param-4': -1.4000382129129179, 'param-5': -16.69598006218125, 'param-6': -8.416848753694026, 'param-7': 4.321351549048657, 'param-8': -19.794183448907404, 'param-9': 16.017054649236798}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:52:52,459] Trial 9 finished with value: 4.030224408029646 and parameters: {'param-0': 0.0, 'param-1': -2.0404262953561254, 'param-2': 1.7173456328074543, 'param-3': -3.835520818279587, 'param-4': -12.28113347819751, 'param-5': -35.87612575809066, 'param-6': -4.541829108064903, 'param-7': -17.79087958397637, 'param-8': -27.71580349905146, 'param-9': -3.9398016471463677}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:52:52,460] Trial 10 finished with value: 1.9631994817050404 and parameters: {'param-0': 0.0, 'param-1': -2.039565618956889, 'param-2': 0.9723904310145102, 'param-3': -2.3064999464712663, 'param-4': 23.576614912671786, 'param-5': -22.60057898220758, 'param-6': -24.733222454390592, 'param-7': 7.732530098678026, 'param-8': -4.662026052710431, 'param-9': 21.033919074926132}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:52:52,462] Trial 11 finished with value: -2.778491696335891 and parameters: {'param-0': 0.0, 'param-1': 2.6751281798844637, 'param-2': -8.215853034538835, 'param-3': -8.291013059372759, 'param-4': 21.42512938705704, 'param-5': -21.199387933110277, 'param-6': -25.184908136425832, 'param-7': -12.643333253651427, 'param-8': -14.773456950284341, 'param-9': 12.284321755877855}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:52:52,463] Trial 12 finished with value: 0.17018850984497835 and parameters: {'param-0': 0.0, 'param-1': 2.820250548622713, 'param-2': 1.136621132622082, 'param-3': -18.70078995723447, 'param-4': -2.5835103508239783, 'param-5': -15.656656204763408, 'param-6': -8.929533505591408, 'param-7': -21.2709434142593, 'param-8': 2.938767183125293, 'param-9': 2.8231308602068026}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:52:52,464] Trial 13 finished with value: -0.20388749212887275 and parameters: {'param-0': 0.0, 'param-1': -0.5802830844558073, 'param-2': -6.350542149128344, 'param-3': 8.952480590579647, 'param-4': 18.749449901557647, 'param-5': -5.167027010135378, 'param-6': -36.692872411681286, 'param-7': -21.415132698691927, 'param-8': -15.231471097579231, 'param-9': -7.035333748090224}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:52:52,466] Trial 14 finished with value: -0.6420108860543067 and parameters: {'param-0': 0.0, 'param-1': 0.4318607173574396, 'param-2': 3.0346406854811008, 'param-3': 12.682998964110276, 'param-4': -3.105492831709576, 'param-5': -3.0641284280965664, 'param-6': -18.41507108243468, 'param-7': -6.504299829394988, 'param-8': -13.57790563957849, 'param-9': 41.392461842213066}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:52:52,468] Trial 15 finished with value: -0.2452110059050675 and parameters: {'param-0': 0.0, 'param-1': 0.3714022746626213, 'param-2': -8.285972464171019, 'param-3': 15.222200357815431, 'param-4': 20.651394305116497, 'param-5': 4.773532839998808, 'param-6': 10.439415555101974, 'param-7': -9.737570017472535, 'param-8': -44.6239604206036, 'param-9': 7.789503427314131}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:52:52,469] Trial 16 finished with value: 2.232056087460125 and parameters: {'param-0': 0.0, 'param-1': 2.2297768246448353, 'param-2': 3.6444783057891073, 'param-3': 3.1381758476708015, 'param-4': 3.857392757435534, 'param-5': -4.113516720963894, 'param-6': 21.209993966533304, 'param-7': 2.6019302779843088, 'param-8': -10.368724859210417, 'param-9': 37.4178189452158}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:52:52,470] Trial 17 finished with value: -0.4812803662504538 and parameters: {'param-0': 0.0, 'param-1': 0.7277089559745242, 'param-2': 9.947966728388455, 'param-3': 2.8432363686265134, 'param-4': -4.552111487391798, 'param-5': -7.504791257110675, 'param-6': -7.2932428962271345, 'param-7': 9.578323851145953, 'param-8': -23.898730304295547, 'param-9': -20.05026212516124}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:52:52,472] Trial 18 finished with value: 0.047321388279854104 and parameters: {'param-0': 0.0, 'param-1': -1.2125147508676193, 'param-2': -2.1591962227639847, 'param-3': 2.315998220031517, 'param-4': 2.9400180870986645, 'param-5': 2.908785719186767, 'param-6': 13.552153657442247, 'param-7': 3.5627531234298147, 'param-8': -2.4280587221328886, 'param-9': 45.05690570072707}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:52:52,473] Trial 19 finished with value: -1.3032210192580413 and parameters: {'param-0': 0.0, 'param-1': 5.716801624948626, 'param-2': 13.308685548548105, 'param-3': -11.619608761773005, 'param-4': -9.120728282140437, 'param-5': -6.090216269562603, 'param-6': 2.9019677989902064, 'param-7': -14.741198176206382, 'param-8': -33.15637558771804, 'param-9': -8.811942191040899}. Best is trial 8 with value: -2.8720687849693523.
[I 2023-03-14 11:52:52,474] Trial 20 finished with value: -3.0656270494144815 and parameters: {'param-0': 0.0, 'param-1': 1.7901250397623123, 'param-2': -7.381056342096175, 'param-3': 16.337029234212295, 'param-4': 2.7769040048445817, 'param-5': -1.0701655822877996, 'param-6': -31.39691693021981, 'param-7': -0.8264426331720074, 'param-8': -13.06776594030977, 'param-9': 24.125703267046717}. Best is trial 20 with value: -3.0656270494144815.
[I 2023-03-14 11:52:52,477] Trial 21 finished with value: -1.145913094787663 and parameters: {'param-0': 0.0, 'param-1': 7.776151262324458, 'param-2': -6.383375695029896, 'param-3': -4.077546497603876, 'param-4': 16.304833190230795, 'param-5': -14.029069051698876, 'param-6': -15.304255817715465, 'param-7': -3.4528726454930165, 'param-8': -12.77855294449489, 'param-9': 29.948503042572653}. Best is trial 20 with value: -3.0656270494144815.
[I 2023-03-14 11:52:52,478] Trial 22 finished with value: -0.6467182827410254 and parameters: {'param-0': 0.0, 'param-1': 3.9054363530918614, 'param-2': -9.657466225589124, 'param-3': 12.875575662416814, 'param-4': -0.6348144603967043, 'param-5': -6.107021719056547, 'param-6': -21.267321455608503, 'param-7': -61.30246260051419, 'param-8': -35.07614598405834, 'param-9': 17.718854991951986}. Best is trial 20 with value: -3.0656270494144815.
[I 2023-03-14 11:52:52,479] Trial 23 finished with value: 2.405729114837162 and parameters: {'param-0': 0.0, 'param-1': 3.9262068176931137, 'param-2': 0.7567369952069747, 'param-3': 1.268906140074403, 'param-4': 5.855977055174861, 'param-5': -6.126436304698849, 'param-6': -21.84383721936448, 'param-7': 15.55764354537473, 'param-8': -5.2208401853942235, 'param-9': 14.68682826810064}. Best is trial 20 with value: -3.0656270494144815.
[I 2023-03-14 11:52:52,481] Trial 24 finished with value: -0.9526879226435986 and parameters: {'param-0': 0.0, 'param-1': 7.101724698514079, 'param-2': 1.2340173907924488, 'param-3': -12.27065120579988, 'param-4': 4.770571155011247, 'param-5': -14.47770613614474, 'param-6': 19.021340821822648, 'param-7': -2.031048819682951, 'param-8': -11.889474701463229, 'param-9': 10.497665426551706}. Best is trial 20 with value: -3.0656270494144815.
[I 2023-03-14 11:52:52,482] Trial 25 finished with value: -1.1518639936432145 and parameters: {'param-0': 0.0, 'param-1': 0.005131629437721941, 'param-2': -12.513806642779745, 'param-3': 16.016339122099197, 'param-4': 9.427955829556709, 'param-5': -8.355269744163927, 'param-6': 14.095940879965639, 'param-7': 16.207078471066154, 'param-8': -24.97044481432077, 'param-9': 30.629309619413462}. Best is trial 20 with value: -3.0656270494144815.
[I 2023-03-14 11:52:52,483] Trial 26 finished with value: -3.134460884130972 and parameters: {'param-0': 0.0, 'param-1': 6.592834121288128, 'param-2': -7.441506435869496, 'param-3': 4.92245071560945, 'param-4': 11.206491681441278, 'param-5': -9.28443768790622, 'param-6': -5.775372709807506, 'param-7': -12.019228638868078, 'param-8': -38.86449629211741, 'param-9': 16.248844798250957}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,484] Trial 27 finished with value: 3.6501567537560287 and parameters: {'param-0': 0.0, 'param-1': 2.549654138102838, 'param-2': -4.962421385204687, 'param-3': -0.6175258917885103, 'param-4': -5.790270348561606, 'param-5': -16.991859528073185, 'param-6': -33.71742830285568, 'param-7': -3.592308171187497, 'param-8': -17.779097604516025, 'param-9': 21.213433218019745}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,485] Trial 28 finished with value: -0.4789539893636976 and parameters: {'param-0': 0.0, 'param-1': -0.8509925725165619, 'param-2': -3.9278102650774933, 'param-3': -3.4457509742058434, 'param-4': 15.441110651553132, 'param-5': -9.447019860255232, 'param-6': -34.25876225233458, 'param-7': -39.45215901163509, 'param-8': -16.96259166716424, 'param-9': 24.367383327602937}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,487] Trial 29 finished with value: 1.1990784520649 and parameters: {'param-0': 0.0, 'param-1': -0.8834132245838298, 'param-2': -13.172459649644402, 'param-3': 15.63285505816691, 'param-4': -10.753980437550162, 'param-5': 6.515376930144768, 'param-6': -24.640816186077913, 'param-7': 14.951236761717453, 'param-8': -13.367560006314662, 'param-9': 14.738018729657114}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,488] Trial 30 finished with value: 4.143212362530454 and parameters: {'param-0': 0.0, 'param-1': 1.0307813407929416, 'param-2': -4.9435600784079305, 'param-3': 7.152550868081796, 'param-4': 7.879449862871283, 'param-5': -1.9388300412021238, 'param-6': 2.6489265804364877, 'param-7': -25.55709555521706, 'param-8': -43.52996333113714, 'param-9': -4.52906901455961}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,490] Trial 31 finished with value: 3.3390752099083416 and parameters: {'param-0': 0.0, 'param-1': 0.9466647414565443, 'param-2': -10.676907728025359, 'param-3': 12.140283980262232, 'param-4': 20.964796086054385, 'param-5': 11.487564133870663, 'param-6': 14.321442218340863, 'param-7': 18.704897731125058, 'param-8': 15.313141412109431, 'param-9': 2.2247562703279016}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,491] Trial 32 finished with value: -2.425339801313445 and parameters: {'param-0': 0.0, 'param-1': 9.013091512310542, 'param-2': -11.408528090777535, 'param-3': 4.629593919034569, 'param-4': 23.404656491339175, 'param-5': -7.436219667473125, 'param-6': -6.5446582784887255, 'param-7': -17.54900227598369, 'param-8': -7.475462317906306, 'param-9': 41.50842145887526}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,492] Trial 33 finished with value: -1.4435452672025417 and parameters: {'param-0': 0.0, 'param-1': 5.463330556826309, 'param-2': -6.413513406378035, 'param-3': 8.161716023744674, 'param-4': 8.19429106219011, 'param-5': -8.756506958248636, 'param-6': -1.7266126937545963, 'param-7': 21.347228107288515, 'param-8': -21.013263767571424, 'param-9': 30.719193037282153}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,494] Trial 34 finished with value: 4.718080694996887 and parameters: {'param-0': 0.0, 'param-1': 1.589042172768071, 'param-2': -4.376895472601664, 'param-3': 16.84459973638984, 'param-4': 13.52784651187497, 'param-5': -10.52361642563401, 'param-6': 26.503058228495917, 'param-7': -15.821502375770578, 'param-8': -35.46514358698461, 'param-9': 18.93887050482715}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,495] Trial 35 finished with value: 4.532316522957611 and parameters: {'param-0': 0.0, 'param-1': 3.966186271929651, 'param-2': -4.8528642597610485, 'param-3': 2.582225267459627, 'param-4': 14.488880810638953, 'param-5': 2.4377309966482557, 'param-6': 5.791207788693214, 'param-7': -61.99840454159726, 'param-8': -48.416120138018904, 'param-9': 71.06084601690955}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,496] Trial 36 finished with value: 0.3366194971726634 and parameters: {'param-0': 0.0, 'param-1': 6.35373859640908, 'param-2': -12.651928669407948, 'param-3': 9.014317097878532, 'param-4': -8.208015923794466, 'param-5': -23.481402808361196, 'param-6': 25.98650823993279, 'param-7': 9.965441235129674, 'param-8': -14.2930980729585, 'param-9': 0.6978618444820768}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,497] Trial 37 finished with value: -1.6210378540684625 and parameters: {'param-0': 0.0, 'param-1': 6.098488452745226, 'param-2': -16.175882211335143, 'param-3': 1.0966054187318868, 'param-4': 10.690419163333225, 'param-5': -14.38244316817621, 'param-6': 7.412444054528649, 'param-7': -0.2508406064023774, 'param-8': -45.19063822215112, 'param-9': -2.528916060955922}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,498] Trial 38 finished with value: -0.7567122280673153 and parameters: {'param-0': 0.0, 'param-1': 5.556343675056299, 'param-2': -8.890982496712958, 'param-3': -6.648188198364554, 'param-4': 3.114566157710442, 'param-5': -16.381976757278437, 'param-6': -8.663294402662444, 'param-7': 19.485002014485445, 'param-8': -25.894668343706982, 'param-9': 13.707689309189817}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,499] Trial 39 finished with value: 0.33972907818342113 and parameters: {'param-0': 0.0, 'param-1': 1.4906947655591125, 'param-2': -12.865456743864492, 'param-3': 3.0822933955022904, 'param-4': 16.746615467467088, 'param-5': -5.53313986017907, 'param-6': 8.387575153138329, 'param-7': -11.92276512221099, 'param-8': 11.163083096852873, 'param-9': -2.3425367260051217}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,500] Trial 40 finished with value: -1.8299997521511608 and parameters: {'param-0': 0.0, 'param-1': 0.08475465774924018, 'param-2': -6.2916454857255815, 'param-3': -13.854218821453973, 'param-4': 8.52979460313832, 'param-5': -12.817475202423154, 'param-6': -19.035366565043056, 'param-7': -2.6313115215011322, 'param-8': -33.637535863857195, 'param-9': 50.25655874668203}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,502] Trial 41 finished with value: 3.6194297091780525 and parameters: {'param-0': 0.0, 'param-1': 1.9470516977860726, 'param-2': -4.641425079142428, 'param-3': 6.136426033471523, 'param-4': 20.294216233063693, 'param-5': -19.08989816796688, 'param-6': 1.4379608733113898, 'param-7': -14.753845470522606, 'param-8': -3.3294453904637464, 'param-9': 38.5057655500579}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,503] Trial 42 finished with value: 2.5014591759487006 and parameters: {'param-0': 0.0, 'param-1': 5.935469745073348, 'param-2': -11.481487558793233, 'param-3': 7.151071021525503, 'param-4': 14.162662679294328, 'param-5': -14.203848866093516, 'param-6': -12.669486456278612, 'param-7': -23.412815341297545, 'param-8': -34.23455464619374, 'param-9': 25.807247384941576}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,504] Trial 43 finished with value: 4.173215105985346 and parameters: {'param-0': 0.0, 'param-1': -0.2753837085702564, 'param-2': -4.708208420148875, 'param-3': 7.668854403660433, 'param-4': 18.513069999582605, 'param-5': -5.882827578083635, 'param-6': -30.24003190726989, 'param-7': 20.48522051145011, 'param-8': -60.273279658668415, 'param-9': 87.89571328043417}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,505] Trial 44 finished with value: 1.847169722157606 and parameters: {'param-0': 0.0, 'param-1': 3.331930612375091, 'param-2': -10.865424872055403, 'param-3': -0.055690575607478365, 'param-4': 10.406526749222415, 'param-5': 2.6230231833256994, 'param-6': -5.693899200997478, 'param-7': 6.442190591256136, 'param-8': 6.285418407597746, 'param-9': 2.33782650883154}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,507] Trial 45 finished with value: 5.261282849921536 and parameters: {'param-0': 0.0, 'param-1': 1.2045651008038032, 'param-2': -10.7154683485313, 'param-3': 1.851515678227475, 'param-4': 0.4682189183140508, 'param-5': -18.314240048251463, 'param-6': -4.863941805050153, 'param-7': 46.713624774104815, 'param-8': -49.58560043222555, 'param-9': 24.52406110499929}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,508] Trial 46 finished with value: -2.1244373959350953 and parameters: {'param-0': 0.0, 'param-1': 5.864348145135281, 'param-2': -13.686330749798525, 'param-3': 4.706353299399403, 'param-4': 8.064102615308606, 'param-5': -38.44576874736448, 'param-6': 6.0554842240631075, 'param-7': -43.52348448727592, 'param-8': -40.45224464451792, 'param-9': 62.87699556497827}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,509] Trial 47 finished with value: -1.123613716955029 and parameters: {'param-0': 0.0, 'param-1': 4.315221999106969, 'param-2': -6.5428640123902255, 'param-3': -6.96394911958874, 'param-4': 3.6257923830902996, 'param-5': -30.049206159278043, 'param-6': -9.75935965318532, 'param-7': -40.98223086546052, 'param-8': -6.089303801693916, 'param-9': 66.48684649484602}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,510] Trial 48 finished with value: -0.3394014294403014 and parameters: {'param-0': 0.0, 'param-1': 5.985944833599383, 'param-2': -10.717615131367658, 'param-3': -2.9855305058545625, 'param-4': -3.206853771849481, 'param-5': -20.947041312392088, 'param-6': -6.263562514039194, 'param-7': -34.02852436076248, 'param-8': -54.261104610430294, 'param-9': 37.370917023991396}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,511] Trial 49 finished with value: -0.5755668207908291 and parameters: {'param-0': 0.0, 'param-1': 5.884150717640708, 'param-2': -11.27383543580115, 'param-3': 1.6934382851061898, 'param-4': 8.733265539153905, 'param-5': -7.2527835813591395, 'param-6': -10.105406933389602, 'param-7': 4.1119241401215305, 'param-8': -27.048533533224003, 'param-9': 30.459325028225663}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,513] Trial 50 finished with value: 0.002732648501080681 and parameters: {'param-0': 0.0, 'param-1': 8.745771354722326, 'param-2': -8.638170359437769, 'param-3': 0.1302850284261936, 'param-4': 32.86215749768911, 'param-5': -20.502456087810074, 'param-6': -1.7307186109503547, 'param-7': -4.274989071404917, 'param-8': -42.584171272945206, 'param-9': 30.173376403993075}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,514] Trial 51 finished with value: 1.9648463443162272 and parameters: {'param-0': 0.0, 'param-1': 5.202816051830332, 'param-2': 4.322365011158237, 'param-3': 19.34213602153627, 'param-4': 14.326884381310123, 'param-5': -23.5833362979602, 'param-6': 9.373053553780963, 'param-7': -48.39435497959147, 'param-8': -4.391630033846155, 'param-9': 16.400885013865874}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,516] Trial 52 finished with value: 3.4229635858181666 and parameters: {'param-0': 0.0, 'param-1': 3.823733617807372, 'param-2': -11.635259881682835, 'param-3': 8.733299876530097, 'param-4': 14.163704421564049, 'param-5': -17.1737858882587, 'param-6': -8.71017383650144, 'param-7': -5.364468846497445, 'param-8': -8.900600208519478, 'param-9': 39.071153191978254}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,517] Trial 53 finished with value: 1.3993851028421667 and parameters: {'param-0': 0.0, 'param-1': 8.482107403792494, 'param-2': -6.867894801203757, 'param-3': 6.3119427637875205, 'param-4': 17.921146149068207, 'param-5': -42.18276509319164, 'param-6': -4.5562195768071945, 'param-7': -50.14464249454794, 'param-8': -20.35950343435139, 'param-9': 83.84399121011131}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,518] Trial 54 finished with value: 0.9513765703765935 and parameters: {'param-0': 0.0, 'param-1': 2.406188702488702, 'param-2': -9.633393452611113, 'param-3': 8.195297353, 'param-4': 18.80447908479873, 'param-5': -32.031298001373976, 'param-6': 10.782246356774735, 'param-7': -35.50647851490283, 'param-8': -48.285153685094016, 'param-9': 67.57910140656253}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,519] Trial 55 finished with value: -2.1816017280899436 and parameters: {'param-0': 0.0, 'param-1': 2.954706344728743, 'param-2': -7.55287557244249, 'param-3': -1.403183396095379, 'param-4': 4.341784362631458, 'param-5': -24.682656189333578, 'param-6': -19.046490898434854, 'param-7': -36.405306909886306, 'param-8': -13.050269029986922, 'param-9': 41.07311645330026}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,520] Trial 56 finished with value: -0.40149755512636975 and parameters: {'param-0': 0.0, 'param-1': 3.5866088201478306, 'param-2': -12.151484433594991, 'param-3': -12.148264164925425, 'param-4': -2.3052166627420974, 'param-5': -19.394579466929518, 'param-6': 20.032984614639943, 'param-7': -69.12605257777267, 'param-8': 16.149511740060547, 'param-9': 84.83003846870363}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,521] Trial 57 finished with value: -0.5793471908897462 and parameters: {'param-0': 0.0, 'param-1': 3.7220420736493374, 'param-2': 0.07358910366919957, 'param-3': -13.8459590842101, 'param-4': 12.56478962108745, 'param-5': -23.068141676165343, 'param-6': 2.508958145208858, 'param-7': -15.803421181256795, 'param-8': -50.23384553662599, 'param-9': 41.67935845331712}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,522] Trial 58 finished with value: 1.5924883802380394 and parameters: {'param-0': 0.0, 'param-1': 3.085793471429639, 'param-2': -9.469237110817204, 'param-3': -5.212688179304106, 'param-4': 1.4231761059741288, 'param-5': -32.44014447825038, 'param-6': 12.359431600931288, 'param-7': -33.36425122351743, 'param-8': -35.40496412132269, 'param-9': 32.46201581459228}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,524] Trial 59 finished with value: 1.9569231942341943 and parameters: {'param-0': 0.0, 'param-1': 1.1243046872353837, 'param-2': -4.143971844466057, 'param-3': 13.221529936144847, 'param-4': -14.764329090715115, 'param-5': -35.511616937940595, 'param-6': 0.14236046683453907, 'param-7': -65.35171968280288, 'param-8': -33.927779484723416, 'param-9': 77.86300136769964}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,525] Trial 60 finished with value: 4.584185380779558 and parameters: {'param-0': 0.0, 'param-1': 0.8031151512522108, 'param-2': -11.688194015792952, 'param-3': 7.506711972428448, 'param-4': -11.337010023340461, 'param-5': -24.903275838154947, 'param-6': 31.584785616852898, 'param-7': -43.03331326406659, 'param-8': -46.73509629438097, 'param-9': 27.88103255135617}. Best is trial 26 with value: -3.134460884130972.
[I 2023-03-14 11:52:52,527] Trial 61 finished with value: -3.7024967182018313 and parameters: {'param-0': 0.0, 'param-1': -1.5783790085883638, 'param-2': 4.419564712851731, 'param-3': 1.8592947852869415, 'param-4': 4.411727358218435, 'param-5': -6.811200421880557, 'param-6': -43.70787707154304, 'param-7': -14.716110244675079, 'param-8': -28.511051234991726, 'param-9': 35.708747313333134}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:52:52,528] Trial 62 finished with value: -0.16937136132681896 and parameters: {'param-0': 0.0, 'param-1': 6.046255509790683, 'param-2': -14.121530769280138, 'param-3': 7.664462877352008, 'param-4': 11.857795148633336, 'param-5': -30.640851786811258, 'param-6': 20.408464559345816, 'param-7': -16.78794654141717, 'param-8': -7.38522188380837, 'param-9': 29.546145225782894}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:52:52,529] Trial 63 finished with value: 0.16504466380504695 and parameters: {'param-0': 0.0, 'param-1': 3.911434040991635, 'param-2': -10.961143693967593, 'param-3': -2.2752816048310613, 'param-4': 28.74189414589152, 'param-5': -25.172392920267352, 'param-6': 6.90425076123536, 'param-7': -24.9596432963433, 'param-8': -41.31498747960013, 'param-9': 72.35366596061579}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:52:52,531] Trial 64 finished with value: -1.6531747253503348 and parameters: {'param-0': 0.0, 'param-1': 2.8556033157794793, 'param-2': -2.499422414068185, 'param-3': -15.75229708241375, 'param-4': 4.6469184137162145, 'param-5': -12.64807383826416, 'param-6': -6.631669534753968, 'param-7': -12.679793042922348, 'param-8': -6.902471441660666, 'param-9': 82.50547085728493}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:52:52,532] Trial 65 finished with value: 0.4813406168180178 and parameters: {'param-0': 0.0, 'param-1': 2.610927564880626, 'param-2': -13.723392423599428, 'param-3': -14.76552966730726, 'param-4': 11.9971460749613, 'param-5': -29.985755791811712, 'param-6': -2.3537847377259027, 'param-7': -67.61139644872165, 'param-8': -12.144457632858632, 'param-9': 71.67397811224728}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:52:52,533] Trial 66 finished with value: -0.3719758239424421 and parameters: {'param-0': 0.0, 'param-1': 8.408761779183386, 'param-2': -1.4187575370889007, 'param-3': -11.099459165650586, 'param-4': 30.505594660665707, 'param-5': -3.1368910646616754, 'param-6': -18.797863800298963, 'param-7': 17.48318396888486, 'param-8': -36.16625735269997, 'param-9': 5.753225103478343}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:52:52,534] Trial 67 finished with value: -0.6806850044923272 and parameters: {'param-0': 0.0, 'param-1': 4.114647420676874, 'param-2': 1.8509339057588647, 'param-3': 5.471167876711725, 'param-4': 16.550970516643204, 'param-5': -16.21615223768432, 'param-6': -15.863760403591108, 'param-7': -22.003333012008476, 'param-8': -30.976201710522197, 'param-9': 24.696640756859665}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:52:52,535] Trial 68 finished with value: -2.0421144402028255 and parameters: {'param-0': 0.0, 'param-1': -1.4099153907576287, 'param-2': 6.587608147363262, 'param-3': -10.845938068054327, 'param-4': -9.140522118162142, 'param-5': -21.825039216688342, 'param-6': 4.519774791328359, 'param-7': -7.681845586645018, 'param-8': -26.324898874278105, 'param-9': 39.208106888847965}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:52:52,537] Trial 69 finished with value: 0.8928730621682152 and parameters: {'param-0': 0.0, 'param-1': -3.2436970362314597, 'param-2': 1.3245764940515485, 'param-3': -11.572998728949692, 'param-4': 1.7794548970201944, 'param-5': -23.261399066709505, 'param-6': -14.659597140704939, 'param-7': -33.12186296399628, 'param-8': -38.11657085725801, 'param-9': 62.07405984524581}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:52:52,538] Trial 70 finished with value: 3.995776158146804 and parameters: {'param-0': 0.0, 'param-1': 1.9289330280648294, 'param-2': -3.2832722821513904, 'param-3': 1.5522265934425974, 'param-4': -3.3794323706385256, 'param-5': -34.45119265176209, 'param-6': 7.875605414935563, 'param-7': -54.28455791709932, 'param-8': -32.757976742473446, 'param-9': 70.57411760551881}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:52:52,540] Trial 71 finished with value: 1.150482656681782 and parameters: {'param-0': 0.0, 'param-1': -0.026055278004466942, 'param-2': -8.428558021262194, 'param-3': -11.54623864249799, 'param-4': 1.765290656711251, 'param-5': -23.482973035606957, 'param-6': -31.296504617201396, 'param-7': -24.74298046238207, 'param-8': -15.268217302876408, 'param-9': 55.456463610953676}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:52:52,541] Trial 72 finished with value: -1.6428969427737077 and parameters: {'param-0': 0.0, 'param-1': 1.3467127760112412, 'param-2': 7.1078063341415, 'param-3': -15.749413627569902, 'param-4': -1.7444442361408221, 'param-5': -25.17598303438454, 'param-6': 5.189507244166677, 'param-7': -13.864384648915085, 'param-8': -35.0579368939648, 'param-9': 73.93811972230861}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:52:52,542] Trial 73 finished with value: 1.8621210646303208 and parameters: {'param-0': 0.0, 'param-1': 2.76848129390282, 'param-2': 10.849910938153396, 'param-3': 1.027852910013575, 'param-4': -17.004658673250994, 'param-5': -3.185381580907496, 'param-6': -36.20662651355284, 'param-7': 6.854135848962372, 'param-8': -15.77446043237525, 'param-9': 23.76510132890904}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:52:52,543] Trial 74 finished with value: 0.3832508101514548 and parameters: {'param-0': 0.0, 'param-1': -2.8886746038330173, 'param-2': 8.616880056174303, 'param-3': -13.337528989038681, 'param-4': -9.4538093562782, 'param-5': -26.992664282328363, 'param-6': 13.991815570114085, 'param-7': 18.577086510165202, 'param-8': -22.500245443989144, 'param-9': 82.0170175108239}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:52:52,545] Trial 75 finished with value: 1.674257290500921 and parameters: {'param-0': 0.0, 'param-1': 6.32721513494014, 'param-2': 4.762698353125781, 'param-3': -18.702246252744338, 'param-4': 22.288142111193324, 'param-5': -24.28198788533924, 'param-6': -13.26988237824748, 'param-7': -22.866583480328615, 'param-8': -29.896010228111948, 'param-9': 7.410911781133578}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:52:52,546] Trial 76 finished with value: -1.0468725115633561 and parameters: {'param-0': 0.0, 'param-1': -1.1610687929452332, 'param-2': 6.559642278817542, 'param-3': -3.5598466943229496, 'param-4': 3.487176494093461, 'param-5': -25.33578906649397, 'param-6': -20.165061111008413, 'param-7': -9.740563204066184, 'param-8': -24.53055399066541, 'param-9': 3.320318187915106}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:52:52,547] Trial 77 finished with value: 4.239897508448774 and parameters: {'param-0': 0.0, 'param-1': -0.18785019765469002, 'param-2': 13.715626286308655, 'param-3': -15.82853615457911, 'param-4': 8.481490542106002, 'param-5': -5.695079996898407, 'param-6': -18.982326288019493, 'param-7': -36.276208212765496, 'param-8': -17.52820750515197, 'param-9': 28.069020499189463}. Best is trial 61 with value: -3.7024967182018313.
[I 2023-03-14 11:52:52,548] Trial 78 finished with value: -5.1657150247361985 and parameters: {'param-0': 0.0, 'param-1': -0.6379163450129948, 'param-2': 1.8159126429309147, 'param-3': 11.535288239524945, 'param-4': -13.404787650407659, 'param-5': -27.35103995100866, 'param-6': -13.352429922820598, 'param-7': -46.08319803074357, 'param-8': -33.67106833131295, 'param-9': 49.344649791676716}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:52:52,550] Trial 79 finished with value: -4.387563821089634 and parameters: {'param-0': 0.0, 'param-1': -2.139560424337783, 'param-2': 12.859799939386875, 'param-3': 3.763327352915539, 'param-4': -9.860565074394493, 'param-5': 4.705403191855275, 'param-6': -27.158640878722274, 'param-7': 11.649194924106894, 'param-8': -3.158042552507453, 'param-9': 36.18694644566973}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:52:52,551] Trial 80 finished with value: -2.1077783523767315 and parameters: {'param-0': 0.0, 'param-1': -1.5053462315868984, 'param-2': 16.857382999748253, 'param-3': -14.306758627539198, 'param-4': 18.59178613146804, 'param-5': 8.171723528578092, 'param-6': -56.4698001958446, 'param-7': -20.67325012116806, 'param-8': -4.915177560091394, 'param-9': 50.26862168864079}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:52:52,553] Trial 81 finished with value: 1.9166016850612984 and parameters: {'param-0': 0.0, 'param-1': -3.858454733916627, 'param-2': 7.894883832247327, 'param-3': -8.592362560696085, 'param-4': 17.92006542960928, 'param-5': -44.327603833441174, 'param-6': -16.66324137759125, 'param-7': 20.819584379083167, 'param-8': -12.500507416602773, 'param-9': 63.17465348007366}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:52:52,554] Trial 82 finished with value: 2.537036780052864 and parameters: {'param-0': 0.0, 'param-1': -5.73656727604585, 'param-2': 0.2438272383147222, 'param-3': 6.319557907324416, 'param-4': -10.284486966914823, 'param-5': -24.52419847114674, 'param-6': -17.62674468088587, 'param-7': -27.749249278389875, 'param-8': -0.5757144511673005, 'param-9': 15.16603800028264}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:52:52,555] Trial 83 finished with value: 0.7609992566997936 and parameters: {'param-0': 0.0, 'param-1': 1.3973502503023632, 'param-2': 4.226024932708107, 'param-3': -3.1882079968997026, 'param-4': -0.6927692038259323, 'param-5': -13.254442460067288, 'param-6': -10.473005317457385, 'param-7': -19.61904883959418, 'param-8': -30.224451151777842, 'param-9': 33.650813729567815}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:52:52,557] Trial 84 finished with value: 1.1919141204107087 and parameters: {'param-0': 0.0, 'param-1': -4.370447514239906, 'param-2': 15.520460762938136, 'param-3': -10.217096400294245, 'param-4': -13.7897949110701, 'param-5': -28.768441119002468, 'param-6': -13.127147107825373, 'param-7': -19.25185792181118, 'param-8': -11.930832063671815, 'param-9': 12.714122609292943}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:52:52,558] Trial 85 finished with value: 5.567073184051192 and parameters: {'param-0': 0.0, 'param-1': 1.2656846733472733, 'param-2': 8.376380567771967, 'param-3': -9.387137366272338, 'param-4': -11.546184121692427, 'param-5': 2.06999843103236, 'param-6': -2.426212095006612, 'param-7': -42.103489558181025, 'param-8': -30.22166646321196, 'param-9': 57.52262962206157}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:52:52,559] Trial 86 finished with value: -1.593024093164151 and parameters: {'param-0': 0.0, 'param-1': -0.8286774214642314, 'param-2': 13.24842775938302, 'param-3': 9.975679318305453, 'param-4': -21.494082511681995, 'param-5': -23.843270998969068, 'param-6': -1.718085785648782, 'param-7': -7.344768618814214, 'param-8': -12.739341990138684, 'param-9': 13.194521835755793}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:52:52,560] Trial 87 finished with value: -1.187663197889515 and parameters: {'param-0': 0.0, 'param-1': -2.1604436747293, 'param-2': 6.3785604962960285, 'param-3': -4.664850861119163, 'param-4': -19.127104117357234, 'param-5': -32.90003465530168, 'param-6': -11.835037735609085, 'param-7': -37.435319933210955, 'param-8': -20.484718351100923, 'param-9': 37.587822173137255}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:52:52,561] Trial 88 finished with value: 2.2698279842419447 and parameters: {'param-0': 0.0, 'param-1': 0.14091357621384581, 'param-2': 14.08854416982232, 'param-3': 6.661342938211895, 'param-4': -6.584994298816191, 'param-5': -3.6758127406666716, 'param-6': -43.70390448058216, 'param-7': -4.964533748754235, 'param-8': -45.62040554772884, 'param-9': 38.00797573305198}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:52:52,562] Trial 89 finished with value: -1.0894970962662252 and parameters: {'param-0': 0.0, 'param-1': -0.1985634757039776, 'param-2': 3.3712220734854412, 'param-3': 4.921320588907179, 'param-4': -10.209553629215883, 'param-5': -30.837734578016857, 'param-6': -14.774871086981932, 'param-7': -29.312476792027333, 'param-8': -21.871900735654663, 'param-9': 73.32950294818255}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:52:52,564] Trial 90 finished with value: -0.7892663375873417 and parameters: {'param-0': 0.0, 'param-1': -3.753221602915115, 'param-2': 5.356661491883411, 'param-3': -9.52023187267767, 'param-4': 8.444391579463392, 'param-5': -21.09898592881474, 'param-6': -25.301337831278396, 'param-7': -16.61914964893672, 'param-8': -38.06819007190808, 'param-9': 61.493101807795114}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:52:52,565] Trial 91 finished with value: 0.7900370067341222 and parameters: {'param-0': 0.0, 'param-1': -3.4587831951038206, 'param-2': 17.02193567194964, 'param-3': -18.642231227536584, 'param-4': -34.056504264521166, 'param-5': -23.79367108902618, 'param-6': -19.896844132123903, 'param-7': 1.2361985605482317, 'param-8': -10.686315752274297, 'param-9': 24.844267682439707}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:52:52,567] Trial 92 finished with value: 1.4357065187484959 and parameters: {'param-0': 0.0, 'param-1': 0.18195713040617, 'param-2': 2.5393199904163097, 'param-3': -0.17469399949462172, 'param-4': -19.038578360599093, 'param-5': -20.746242052748514, 'param-6': 3.1545494522133524, 'param-7': -17.178192407398377, 'param-8': -18.675670423702257, 'param-9': 46.12234354023357}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:52:52,568] Trial 93 finished with value: 1.332182331440031 and parameters: {'param-0': 0.0, 'param-1': 2.0370533136664157, 'param-2': -4.350851107825646, 'param-3': 5.008909436261391, 'param-4': 7.7871162592198, 'param-5': -39.6520378555507, 'param-6': 20.303537240732823, 'param-7': -8.265465460834697, 'param-8': 12.739453040397166, 'param-9': 25.274375614492754}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:52:52,569] Trial 94 finished with value: 0.40226422472793383 and parameters: {'param-0': 0.0, 'param-1': -6.1194773685811565, 'param-2': 11.018348416053119, 'param-3': 3.553352741203966, 'param-4': -27.42635063634117, 'param-5': -17.82842579138267, 'param-6': -3.2035363285535894, 'param-7': 21.356894974181685, 'param-8': 19.16180809359338, 'param-9': -30.803547439789}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:52:52,570] Trial 95 finished with value: -1.2565270265317818 and parameters: {'param-0': 0.0, 'param-1': -3.101331149628095, 'param-2': 3.973159512201793, 'param-3': -21.921873775272836, 'param-4': -8.413896914481409, 'param-5': -16.63505371381693, 'param-6': 2.655747153563091, 'param-7': 5.587648258267876, 'param-8': -24.450855339040643, 'param-9': 43.025999162573214}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:52:52,571] Trial 96 finished with value: -0.38334157874526686 and parameters: {'param-0': 0.0, 'param-1': -0.5462971040180911, 'param-2': 13.371927980411712, 'param-3': 22.383131644870637, 'param-4': -37.67515041764983, 'param-5': -25.0370079797484, 'param-6': -39.523577004865786, 'param-7': -15.108133668102809, 'param-8': -61.32390742758892, 'param-9': 37.91264949828205}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:52:52,572] Trial 97 finished with value: 2.143693476406879 and parameters: {'param-0': 0.0, 'param-1': -3.4477349322443356, 'param-2': 12.992862632623805, 'param-3': 2.0464340812718973, 'param-4': -17.817610598207448, 'param-5': -45.484826476283196, 'param-6': -2.097434051166978, 'param-7': -37.46823986652984, 'param-8': 8.68460676366324, 'param-9': 27.579492672096194}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:52:52,573] Trial 98 finished with value: -0.4721133717660624 and parameters: {'param-0': 0.0, 'param-1': -0.6685362691693424, 'param-2': 11.389123387056546, 'param-3': 8.155258288837707, 'param-4': -24.117842099798604, 'param-5': -10.192299404151399, 'param-6': -14.076152750684429, 'param-7': -32.153057098472516, 'param-8': -4.206544169711663, 'param-9': 10.10880727646743}. Best is trial 78 with value: -5.1657150247361985.
[I 2023-03-14 11:52:52,575] Trial 99 finished with value: -1.3476625107916274 and parameters: {'param-0': 0.0, 'param-1': 2.2797704200583198, 'param-2': 15.633170735927983, 'param-3': -7.496188700385282, 'param-4': 4.282449410083096, 'param-5': -26.11404793929163, 'param-6': -10.221254746173926, 'param-7': -39.42921570589232, 'param-8': -18.701178818045364, 'param-9': 2.4727491512137334}. Best is trial 78 with value: -5.1657150247361985.
FrozenTrial(number=78, state=TrialState.COMPLETE, values=[-5.1657150247361985], datetime_start=datetime.datetime(2023, 3, 14, 11, 52, 52, 547624), datetime_complete=datetime.datetime(2023, 3, 14, 11, 52, 52, 548543), params={'param-0': 0.0, 'param-1': -0.6379163450129948, 'param-2': 1.8159126429309147, 'param-3': 11.535288239524945, 'param-4': -13.404787650407659, 'param-5': -27.35103995100866, 'param-6': -13.352429922820598, 'param-7': -46.08319803074357, 'param-8': -33.67106833131295, 'param-9': 49.344649791676716}, user_attrs={}, system_attrs={'cma:generation': 7, 'cma:n_restarts': 0}, intermediate_values={}, distributions={'param-0': FloatDistribution(high=0.0, log=False, low=0.0, step=None), 'param-1': FloatDistribution(high=10.0, log=False, low=-10.0, step=None), 'param-2': FloatDistribution(high=20.0, log=False, low=-20.0, step=None), 'param-3': FloatDistribution(high=30.0, log=False, low=-30.0, step=None), 'param-4': FloatDistribution(high=40.0, log=False, low=-40.0, step=None), 'param-5': FloatDistribution(high=50.0, log=False, low=-50.0, step=None), 'param-6': FloatDistribution(high=60.0, log=False, low=-60.0, step=None), 'param-7': FloatDistribution(high=70.0, log=False, low=-70.0, step=None), 'param-8': FloatDistribution(high=80.0, log=False, low=-80.0, step=None), 'param-9': FloatDistribution(high=90.0, log=False, low=-90.0, step=None)}, trial_id=78, value=None)
```

</details>

Reply:
Ah, I understand. Let me resolve this.